src/pymor/la/numpysolvers.py

```diff
@@ -481,15 +481,15 @@ def apply_inverse(matrix, U, options=None):
         else:
             if hasattr(matrix, 'factorization'):
                 for i, UU in enumerate(U):
-                    R[i] = matrix.factorization.solve(UU)
```

Comment:
Yes, I am sure there is a problem and I explained it in my description of this pulll request.